sdk/highlight-run/src/sdk/observe.ts

```diff
@@ -221,6 +221,14 @@ export class ObserveSDK implements Observe {
 		errorMsg: ErrorMessage,
 		payload?: { [key: string]: string },
 	) {
+		try {
+			if (errorMsg.payload) {
+				payload = {
+					...(payload ? payload : {}),
+					...JSON.parse(errorMsg.payload),
```

Comment:
nit: flip the order?
If there's duplicate keys between the maps, I'd imagine the original payload has a higher precedence and should be kept.